components/event_dispatcher.rst

```diff
@@ -135,7 +135,7 @@ A call to the dispatcher's ``addListener()`` method associates any valid
 PHP callable to an event::
 
     $listener = new AcmeListener();
-    $dispatcher->addListener('acme.action', array($listener, 'onFooAction'));
+    $dispatcher->addListener('foo.action', array($listener, 'onFooAction'));
```

Comment:
By the way the example I given is perfectly valid, see https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/FOSUserEvents.php#L26 as an example :)